deploy/backplane/cee/30-cee-pcap-collector.Role.yml

```diff
@@ -0,0 +1,18 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: backplane-cee-pcap-collector
+  namespace: openshift-backplane-managed-scripts
+rules:
+# can create cred secret for mustgathers
+# Enables https://github.com/openshift/managed-scripts/blob/main/scripts/CEE/pcap-collector/metadata.yaml, which requires users to be able to create/get/delete the pcap-collector-creds secret in the managed scripts ns
+- verbs:
+    - "get"
```

Comment:
Let's remove the "get" permission, or user will be able to see the secret created by others.

Reply:
And only keep the "create" permission, here is an example for must-gather operator. https://github.com/openshift/managed-cluster-config/blob/master/deploy/backplane/srep/10-srep-mustgather.Role.yml#L13

Reply:
:heavy_check_mark: